storage/src/metadata/mod.rs

```diff
@@ -23,15 +23,11 @@
 //! _To ensure the integrity of the data, a CRC32 checksum is appended to the end of the blob.
 //! This ensures that partial writes are detected before any data is relied on._
 //!
-//! _In the unlikely event that the current timestamp since the last `sync` is unchanged (as measured
```

Comment:
On Windows, SystemTime is backed by a FILETIME whose native resolution is 100 ns. When creating a time that differs from the Unix epoch by only 1 ns it is rounded down to the closest 100-ns tick.

This means this trick isn't really guaranteed to work (it was jank anyways).